src/components/FeatureViews/Search/SearchResults.js

```diff
@@ -43,26 +44,31 @@ const SearchResults = (props) => {
         course: 4,
     });
     const [currentPage, setCurrentPage] = useState({
```

Comment:
Since page is something that's given to us by the backend and we're storing it in redux, we should consider getting rid of this state variable and only update the page through redux. otherwise we'd need to update it both in this component (SearchResults) and in Redux.